e2e/updatecli.d/success.d/helmchart.yaml

```diff
@@ -42,6 +42,7 @@ targets:
     spec:
       name: chart/epinio
       file: Chart.yaml
+      files: []
```

Comment:
Is this line needed?

Reply:
It was before the addition of the missing jsonschema annotations you suggested below, pushing a fix.